doc/sphinx/proof-engine/ltac.rst

```diff
@@ -422,7 +422,11 @@ Selectors can also be used nested within a tactic expression with the
       range_selector ::= @natural - @natural
       | @natural
 
-   Applies :token:`ltac_expr3` to the selected goals.
+   Applies :token:`ltac_expr3` to the selected goals.  (At the beginning of a
+   sentence, use the form :n:`@selector: @tactic` rather than :n:`only @selector: @tactic`.
+   In the latter, the :opt:`Default Goal Selector` (commonly set to :n:`1:`)
```

Comment:
```suggestion
   In the latter, the :opt:`Default Goal Selector` (by default set to :n:`1:`)
```

Reply:
That was my first thought, but I don't like repeating "Default ... default" in the sentence.

Reply:
"commonly" doesn't mean the same thing, that's more important than avoiding repetition